yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -388,7 +388,9 @@ func checkAndHandleSnapshotComplete(status *dbzm.ExportStatus, progressTracker *
 	if err != nil {
 		return false, fmt.Errorf("failed to rename dbzm exported data files: %v", err)
 	}
-	displayExportedRowCountSnapshot()
+	if source.DBType != YUGABYTEDB {
```

Comment:
Can you change this to a check on the exporter role? Tomorrow if we decided to support YB as a source, this will break. I think we can have 
if exporterRole != TARGET_DB_EXPORTER { display...()}